cli/azd/pkg/config/manager_test.go

```diff
@@ -39,3 +42,75 @@ func Test_SaveAndLoadEmptyConfig(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, existingConfig)
 }
+
+func Test_GetUserConfigDir(t *testing.T) {
+
+	// Setup temp directory for use in tests, delete after creation to validate
+	// folder creation
+	testDir := t.TempDir()
+	os.Remove(testDir)
```

Comment:
Surprised to see the linter not catch this.. `os.Remove()` will return `error` if the removal fails